### PR TITLE
Added showJoinMessage and showLeaveMessage on the IRC side.

### PR DESCRIPTION
**Pull request by xforever1313**
The telegram side of teleirc has the option to show join/leave messages from the IRC side.  However, the IRC side does not have the same.  This pull request adds that.

Disclaimer: I did not formally test it.

### CODE REVIEW

**Comment by jflory7** [comment]
@xforever1313 Thanks for submitting! Hopefully @repkam09 or @thenaterhood can take a look here. Have you had a chance to test any of your changes?

File: teleirc.js [hunk 1]
```diff
@@ -106,10 +106,15 @@ tgbot.on('message', function (msg) {
         if (msg.text === undefined) {
 
             // Check if this message is a new user joining the group chat:
-            if (msg.new_chat_member) {
+            if (msg.new_chat_member  && config.irc.showJoinMessage) {
```
Comment (thenaterhood) [inline teleirc.js]: Suuuuper nitpicky, but you've got an extra space before your "&&".

File: teleirc.js [hunk 2]
```diff
@@ -106,10 +106,15 @@ tgbot.on('message', function (msg) {
...
                 let username = msg.new_chat_member.username;
                 let first_name = msg.new_chat_member.first_name;
                 ircbot.say(config.channel, "New user " + first_name + " ( @" + username + ") has joined the Telegram Group!");
```
Comment (thenaterhood) [inline teleirc.js]: I feel like "New user" (and below, "user") prefacing the username of the join or leave is redundant.

File: teleirc.js [hunk 3]
```diff
@@ -106,10 +106,15 @@ tgbot.on('message', function (msg) {
...
+            // Check if this message is a user leaving the telegram group chat:
+            } else if (msg.left_chat_member && config.irc.showLeaveMessage) {
+                let username = msg.left_chat_member.username;
+                let first_name = msg.left_chat_member.first_name;
+                ircbot.say(config.channel, "User " + first_name + " ( @" + username + ") has left the Telegram Group.");
```
Comment (thenaterhood) [inline teleirc.js]: Again, super nit: the space between the "(" and the "@" doesn't seem necessary.

**Comment by xforever1313** [comment]
Ready for review.  Tested it on my machine and it seems to work.

**Comment by thenaterhood** [comment]
I'm cool with it. I'll leave it open for comment for now though.

**Comment by jflory7** [comment]
Perfect! This all sounds great to me. Thanks for contributing, @xforever1313!

:clapper:

**Pull request merged**